sdk/schemaregistry/schema-registry-avro/src/schemaRegistryAvroSerializer.ts

```diff
@@ -190,6 +188,12 @@ export class SchemaRegistryAvroSerializer {
       throw new Error("Schema must have a name.");
     }
 
+    if (!this.schemaGroup) {
+      throw new Error(
+        "Schema group must have been specified in the constructor options when the client was created."
```

Comment:
Probably worth saying "must have been specified *in order to serialize*"  here.

Reply:
Done.